tests/blackbox/stratis_cli_cert.py

```diff
@@ -115,6 +116,22 @@ def setUp(self):
         time.sleep(1)
         exec_command(["udevadm", "settle"])
 
+    def test_drop_permissions(self, command_line, permissions, exp_stdout_empty):
```

Comment:
Prepend an ```'_'``` since this method is private. (There are other methods that should have an ```'_'``` but don't, but I'm ignoring that for now.)